pkgs/development/python-modules/py-multihash/default.nix

```diff
@@ -33,11 +33,18 @@ buildPythonPackage rec {
     varint
   ];
 
+  # Remove overly strict base58 version check, as the version in nixpkgs is currently too new. The required version has been incremented upstream, however has not made it into any release yet.
+  postPatch = ''
+    sed -i "s/base58>=.*/base58',/" setup.py
```

Comment:
Version 1.0.0 of this package was released and it has a more lenient base58 pin.

Reply:
Where did you find that? I can only seem to find [version 0.2.3](https://pypi.org/project/py-multihash/) on pypi.

Reply:
https://github.com/multiformats/py-multihash

Reply:
pypi is on 0.2.3 and the version at https://github.com/multiformats/py-multihash is 1.0.0.

Reply:
My bad, i misread that. The `homepage` section on pypi links to another github repository, with no tags present. I've switched to the Github releases from the repo @Luflosi linked :)